packages/harmony/src/components/input/SelectablePill/SelectablePill.tsx

```diff
@@ -57,6 +80,11 @@ export const SelectablePill = (props: SelectablePillProps) => {
       paddingInline: theme.spacing.l,
       boxShadow: theme.shadows.near
     }),
+    ...(size === 'oversized' && {
+      height: 49,
```

Comment:
this is quite specific, should we use a standard spacing value here instead?

Reply:
yeah weird right? i was trying to copy exactly the one use-case it's used, but sure can map that to the closest standard spacing.